pytket/extensions/aqt/multi_zone_architecture/circuit_routing/partition_routing.py

```diff
@@ -65,9 +65,8 @@ def get_routed_circuit(self) -> MultiZoneCircuit:
         )
         for old_place, new_place in self.placement_generator(depth_list):
             if self._settings.debug_level > 0:
-                print("-------")
                 for zone in range(self._arch.n_zones):
-                    changes_str = ", ".join(
+                    ", ".join(
```

Comment:
This string now unused -- delete?

Reply:
I am not sure why we are deleting all the `print()` calls. They are presumably for debugging.

Reply:
Ruff suggested that we should remove the print / replace it with some special log. I think in our case it is fine to have the prints. I have added them back in.